api/tools/src/cron/expire-users.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Start hit the reporting snapshot & upload cron endpoint (insert query mode)
+# locally.
+curl -X GET 'http://localhost:8001/v1/cron/expireNonCompliantUsers' \
```

Comment:
Were you able to run this?  Then I run *this script specifically* I see this result:
```curl: (52) Empty reply from server```
and the server outputs
```Debugger failed to attach: handshake failed - received >GET /v1/cron/e< - expected >JDWP-Handshake<```

When I run the command in this script (outside of the script) it succeeds.  (The cron endpoint works fine)